src/main/java/com/plivo/api/PlivoAPIService.java

```diff
@@ -66,6 +76,25 @@
 
 public interface PlivoAPIService {
 
+  // Address
+  @POST("Account/{authId}/Verification/Address/")
+  Call<AddressCreateResponse> addressCreate(@Path("authId") String authId,
+                                            @Body AddressCreator addressCreator);
+
+  @GET("Account/{authId}/Verification/Address/")
+  Call<ListResponse<Address>> addressList(@Path("authId") String authId,
+                                          @QueryMap Map<String, Object> addressListRequest);
+
+  @GET("Account/{authId}/Verification/Address/{addressId}/")
+  Call<Address> addressGet(@Path("authId") String authId, @Path("addressId") String addressId);
+
+  @POST("Account/{authId}/Verification/Address/{addressId}/")
+  Call<AddressUpdateResponse> addressUpdate(@Path("authId") String authId,
```

Comment:
AddressUpdate API not updating address.

`Address.updater(addressId).addressLine1("139").firstName("Bruce 3").update();`

Reply:
as discussed we can take this up in the next release. I see similar issue with python as well. The api returns success but update is not happening correctly even for other implementations